tools/alpine/Dockerfile

```diff
@@ -93,6 +93,8 @@ RUN set -e && \
 
 FROM alpine:3.13
 
+ARG TARGETARCH
```

Comment:
What populates `TARGETARCH`?
I also wonder whether we should rename this to be `IMAGE_SUFFIX` as it better described it's purpose, especially as the arch != what you'd expect from `uname -m`

Reply:
Ugh I should read commit messages better. I see where this is populated now :joy: